spec/requests/objects_controller_file_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ObjectsController, type: :request do
+  let(:prefixed_druid) { 'druid:bj102hs9687' }
+  let(:bare_druid) { 'bj102hs9687' }
+  let(:pres_obj) { create(:preserved_object) }
+
+  describe 'GET #file' do
+    context 'when object exists' do
+      context 'when druid is prefixed' do
+        context 'when no version param specified' do
+          context 'when file is present' do
+            context 'manifest' do
```

Comment:
there is a lot of nesting here, I know.  Suggestions welcome.

Reply:
I don't think there is a problem with the nesting, but I would change to "when the type is manifest"

Reply:
done.